### ISSUE TITLE
Bluetooth

### ISSUE THREAD

**Issue by Simoffen** [ahg95/UnityBluetoothConnection#1]
I can only find example how to use Serial/USB 
, not Bluetooth. 
Am I wrong?

**Comment by ahg95** [comment]
You are right. In the beginning I wanted to use Bluetooth, but found a workaround with using serial input. Repository name is misleading

**Comment by Simoffen** [comment]
Ahhh tx. If you find a way of using BLE from Micro:bit to Unity, please
give me a hint.
Cheers




lör 2 mars 2024 kl. 22:41 skrev Ansgar Johannes Glup <
***@***.***>:

> You are right. In the beginning I wanted to use Bluetooth, but found a
> workaround with using serial input. Repository name is misleading
>

**Comment by ahg95** [comment]
If you have two microbits you can connect one microbit to the pc and use the serialization script that I wrote to read the data that is coming from Unity. You can then connect the second microbit to the first one by radio. Connecting two microbits by radio is very straight forward. Then you can transfer the serialized data from Unity to the connected microbit and then to the second microbit using Radio. Hope this helps qnd is applicable to your situation

**Comment by Simoffen** [comment]
I have to buy one more i guess.



sön 3 mars 2024 kl. 14:19 skrev Ansgar Johannes Glup <
***@***.***>:

> If you have two microbits you can connect one microbit to the pc and use
> the serialization script that I wrote to read the data that is coming from
> Unity. You can then connect the second microbit to the first one by radio.
> Connecting two microbits by radio is very straight forward. Then you can
> transfer the serialized data from Unity to the connected microbit and then
> to the second microbit using Radio. Hope this helps qnd is applicable to
> your situation
>
